tests/test_uwsgi_plugin.py

```diff
@@ -0,0 +1,62 @@
+from testing.sandbox import install_fake_uwsgi
+install_fake_uwsgi()
```

Comment:
damn, maybe that was an old version -- I swore I used uwsgi at some point, maybe I moved it to raw sockets